src/odemis/model/_core.py

```diff
@@ -128,6 +128,8 @@ def _getChildren(root):
     ret = {root}
     for child in root.children.value:
         ret |= _getChildren(child)
+    for dep in root.dependences.value:
```

Comment:
I don't think this is needed. The point is to get all the components created (and which are useful). So the (delegated) children should be sufficient.